frontend/src/Explorer/Types/Actions.purs

```diff
@@ -71,6 +71,8 @@ data Action
     -- clock
     | SetClock DateTime
     | UpdateClock
+    -- socket-io workaround
+    | ReloadSignal
```

Comment:
Just a tiny thing: Lets rename `ReloadSignal` to `Reload`. It might be confusing, because it is an Action, not a Signal here.